### PR TITLE
Fix a bug where the Device Status escape sequence was saved in the scrollback buffer

### PR DESCRIPTION
**Pull request by austinjones**

### CODE REVIEW

File: tab-daemon/src/service/pty/scrollback.rs [hunk 1]
```diff
@@ -98,6 +98,15 @@ impl ScrollbackBuffer {
     }
 
     pub fn push(&mut self, mut chunk: OutputChunk) {
+        // replace ESC [ 6n, Device Status Report
+        // tbis sequence is echoed as keyboard characters,
```
Comment (casonadams) [inline tab-daemon/src/service/pty/scrollback.rs:102]: type on in `tbis` maybe `this`?

**Comment by casonadams** [comment]
This seems to fix the garbage output I as seeing when I was using `vi` in a `tab`
[Issue #238](https://github.com/austinjones/tab-rs/issues/238)

**Comment by casonadams** [comment]
I just reproduced the issue getting `11;rgb:2828/2828/2828`

Create `tab 0`
Open `vi`
`ctrl+T` 
Hit `ESC`
Open `tab 0`

File: tab-daemon/src/service/pty/scrollback.rs (hunk 1 above)
Comment (austinjones) [reply tab-daemon/src/service/pty/scrollback.rs:102]: Ha, yep.  Haven't had a chance to self-review this yet.

**Comment by austinjones** [comment]
Yeah, this change introduces some bugs.  Needs more work.

**Comment by austinjones** [comment]
The second sequence being printed here is an Operating System Command escape sequence.  Apparently if a ? mark is given, then the terminal driver will echo via stdin.  That's where the colors are coming from.

https://www.xfree86.org/current/ctlseqs.html

**Pull request merged**